### ISSUE TITLE
The name is occupied

### ISSUE THREAD

**Issue by eliaspanagiotidis** [claytongulick/pgproxy#1]
I just found this, and I think the idea is brilliant and the tool is extremely convenient for all users of PLV8. However the name (pgproxy) is already used by a postgresql extension, so it may cause confusion. Maybe plv8proxy would be more appropriate.

**Comment by claytongulick** [comment]
plv8proxy is a good idea - except that plv8 might be ditching v8?

What do you think of pljsproxy?

**Comment by eliaspanagiotidis** [comment]
Yes you are right! Most likely v8 will be abandoned. pljsproxy is perfect.
